src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static String password;
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc_intro";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD_PATH = "src/main/resources/password.txt";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        try (BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(PASSWORD_PATH))) {
+            password = bufferedReader.readLine();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);

```

Comment:
Consider using a more informative message for the exception that includes the path to the file that couldn't be read.